package.json

```diff
@@ -22,11 +22,11 @@
   "dependencies": {
     "axios": "0.27.2",
     "rimraf": "3.0.2",
-    "tar": "6.1.11"
+    "tar": "6.1.11",
+    "zod": "4.0.5"
```

Comment:
I'm not really sure why `zod` is here. I don't think it should be, since it's a runtime dependency. Instead, it should be imported like all of the other dependencies using `deps.ts`.